src/Appwrite/Enum/MessageStatus.php

```diff
@@ -2,26 +2,51 @@
 
 namespace Appwrite\Enum;
 
-enum MessageStatus: string
+// TODO: Convert to backed enum after upgrading to PHP 8.1
+// enum MessageStatus: string
+// {
+//     /**
+//      * Message that is not ready to be sent
+//      */
+//     case Draft = 'draft';
+//     /**
+//      * Scheduled to be sent for a later time
+//      */
+//     case Scheduled = 'scheduled';
+//     /**
+//      * Picked up by the worker and starting to send
+//      */
+//     case Processing = 'processing';
+//     /**
+//      * Sent without errors
+//      */
+//     case Sent = 'sent';
+//     /**
+//      * Sent with some errors
+//      */
+//     case Failed = 'failed';
+// }
+
```

Comment:
Let's remove the comment and add the code back once on 8.2

Reply:
Removed